go.mod

```diff
@@ -30,6 +28,8 @@ require (
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-beta.2
+	github.com/cosmos/cosmos-sdk v0.0.0-00010101000000-000000000000
```

Comment:
[question] this version number seems odd, is it intended?

Reply:
it's certainly not. I am still investigating it

Reply:
this is probably coming from the conflicting cosmos-sdk and celestia-core versions.
I guess it will be fixed when we merge the cosmos-sdk QGB branch to the default one and we rebase on master. 
Will merge for now and take a look on it later